src/app/GitCommands/Git/GitModule.cs

```diff
@@ -3515,28 +3515,7 @@ public string GetFileText(ObjectId id, Encoding encoding, bool stripAnsiEscapeCo
         }
 
         public MemoryStream? GetFileStream(string blob)
```

Comment:
Maybe add cancellation token (affects called functions too of course)

Reply:
Cancellation added